Monika After Story/game/script-topics.rpy

```diff
@@ -546,6 +546,32 @@ label monika_selfesteem:
             m 3hub "I'll always root for you, [player]!"
     return "derandom"
 
+init 5 python:
```

Comment:
Probably doesn't matter too much, but I thought new topics were supposed to go near the bottom. Maybe it's just me, but I think that helps with organization.

Reply:
Ah, I see, I'll move it to the bottom.